src/Repositories/Products/ConfigurableProductRepository.php

```diff
@@ -497,15 +388,15 @@ public function createProduct($requestData, $imageZipName, $product)
                                             $data['width'] = (string)$csvData[$i]['width'];
                                             $data['height'] = (string)$csvData[$i]['height'];
                                             $data['depth'] = (string)$csvData[$i]['depth'];
-                                            $data['status'] = (string)$csvData[$i]['status'];
                                             $data['attribute_family_id'] = $attributeFamilyData->id;
                                             $data['short_description'] = (string)$csvData[$i]['short_description'];
                                             $data['sku'] = (string)$csvData[$i]['sku'];
                                             $data['name'] = (string)$csvData[$i]['name'];
                                             $data['weight'] = (string)$csvData[$i]['super_attribute_weight'];
                                             $data['status'] = (string)$csvData[$i]['status'];
+                                            $data['type'] = (string)$csvData[$i]['type'];
```

Comment:
not needed. We get the following error message
Please ensure the product type 'variant' is configured in your application.